### PR TITLE
Update winapi to 0.3

### PR DESCRIPTION
**Pull request by Eijebong**

### CODE REVIEW

**Comment by Eijebong** [comment]
The failures are due to the lockfile format changing

**Comment by ruuda** [comment]
You can fix the failures by generating the lock file with an older version of Cargo.

**Comment by Eijebong** [comment]
Should be ok now

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "thread-id"
-version = "3.2.0"
+version = "3.2.1"
```
Comment (ruuda) [inline Cargo.toml]: Could you please avoid touching this? I bump the version just before making the release, not after making a change.

**Review by ruuda — APPROVED** [review]
Thanks for making the build pass! LGTM, but could you please avoid bumping the version?

**Comment by ruuda** [comment]
Thanks!

**Pull request merged**